serverless.yml

```diff
@@ -4,7 +4,7 @@ provider:
   name: aws
   runtime: nodejs6.10
   stage: v1
-  region: us-west-2
+  region: ${opt:AWS_REGION}
```

Comment:
Have you tested this ? Shouldn't be ${env: opt:AWS_REGION} instead ?

Reply:
https://github.com/serverless/serverless/issues/3947

Reply:
We can change the approach if you like.  I used a command-line argument, so the choice is explicit instead of default.

Reply:
We can change the approach if you like.  I used a command-line argument, so the choice is explicit instead of default.  We tend to want all aspects explicit in our code, so this is too.